app/views/pages/privacy.html.erb

```diff
@@ -1,239 +1,261 @@
+          "request that the processing of your personal data is restricted in certain circumstances — for example, where accuracy is contested",
+        ]
+      } %>
+
+      <p class="govuk-body">If your personal data is processed on the basis of consent, you have the
+        right to:
+      </p>
+
+      <%= render "govuk_publishing_components/components/list", {
+        visible_counters: true,
+        items: [
+          "request a copy of any personal data you have provided, and for this to be provided in a structured, commonly used and machine-readable format",
+        ]
+      } %>
+
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Changes to this notice",
+        margin_bottom: 4,
+        font_size: "m",
+      } %>
+
+      <p class="govuk-body">
+        We may change this privacy notice at our discretion at any time. In that
+        case the ‘Last updated’ date at the top of this page will also change. Any
+        changes to this privacy notice will apply to you and your data
+        immediately. If these changes affect how your personal data is processed,
+        GDS will take reasonable steps to make sure you know.
+      </p>
+
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Questions and complaints",
+        margin_bottom: 4,
+        font_size: "m",
+      } %>
+
+      <p class="govuk-body">
+        The data controller for your personal data is the Cabinet Office. Contact
+        the  GDS Privacy Team if you have any questions or think that
+        your personal data has been misused or mishandled.
+      </p>
+
+      <p class="govuk-body">
+        Email: <%= mail_to 'gds-privacy-office@digital.cabinet-office.gov.uk', nil, class: 'govuk-link' %>
+      </p>
+
+      <p class="govuk-body">
+        The contact details for our Data Protection Officer are:
+      </p>
+
+      <div class="contact">
+        <p class="govuk-body">
+          Data Protection Officer<br />
+          <%= mail_to 'DPO@cabinetoffice.gov.uk', nil, class: 'govuk-link' %><br />
+          Cabinet Office<br />
+          70 Whitehall<br />
+          London <br />
+          SW1A 2AS
+        </p>
+      </div>
+
+      <p class="govuk-body">
+        The Data Protection Officer provides independent advice and monitoring of our use of personal information.
+      </p>
+
+      <p class="govuk-body">
+        If you have a complaint, you can also contact
+        the <%= link_to 'Information Commissioner', 'https://ico.org.uk/', class: 'govuk-link' %>, who
+        is an independent regulator set up to uphold information rights.
+      </p>
+
+      <div class="contact">
+        <p class="govuk-body">
+          Information Commissioner’s Office<br />
+          <%= mail_to 'casework@ico.org.uk', nil, class: 'govuk-link' %><br />
+          0303 123 1113</br>
```

Comment:
Slight typo that probably makes no different to the rendered page, but it'd be nice to fix:

```suggestion
          0303 123 1113<br />
```